src/LINEBot.php

```diff
@@ -95,6 +95,50 @@ public function getProfile($userId)
         return $this->httpClient->get($this->endpointBase . '/v2/bot/profile/' . urlencode($userId));
     }
 
+    /**
+     * Gets the list of User IDs of users who have added your LINE Official Account as a friend.
+     * These users' IDs won't be included in the obtained list of user IDs:
+     * ・ Users who blocked the target LINE Official Account after adding it as a friend.
+     * ・ Users who haven't consented to their profile information being obtained.
```

Comment:
Please remove unicode characters
```suggestion
     * - Users who blocked the target LINE Official Account after adding it as a friend.
     * - Users who haven't consented to their profile information being obtained.
```
Or remove them completely.